internal/pkg/template/templates/overrides/yamlpatch/README.md

```diff
@@ -0,0 +1,17 @@
+# Overriding Copilot generated CloudFormation templates with YAML Patches
+
+The file `cfn.patches.yml` contains a list of YAML/JSON patches to apply to
+your template before AWS Copilot deploys it.
+
+Learn about patching YAML [here](https://www.rfc-editor.org/rfc/rfc6902).
+Only the [`add`](https://www.rfc-editor.org/rfc/rfc6902#section-4.1),
+[`remove`](https://www.rfc-editor.org/rfc/rfc6902#section-4.2), and
+[`replace`](https://www.rfc-editor.org/rfc/rfc6902#section-4.3)
+operations are supported by Copilot.
+
+Patches are applied in the order specified in the file.
+
```

Comment:
```suggestion
## Further material

To view examples and an explanation of how YAML patches work, check out the [documentation](https://aws.github.io/copilot-cli/docs/developing/overrides/yamlpatch).

```

Reply:
see what you think - i'll highlight our page over the RFC?